rocAL/docs/examples/image_processing/decoder.py

```diff
@@ -0,0 +1,47 @@
+
+from tkinter import W
+from amd.rocal.pipeline import pipeline_def
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import matplotlib.gridspec as gridspec
+import matplotlib.pyplot as plt
+#%matplotlib inline
+
+seed = 1549361629
+image_dir = "/home/rajy/work/MIVisionX-data/rocal_data/images_jpg/labels_folder"
```

Comment:
@rrawther Need to remove hardcoded path. Needs to be an arg

Reply:
fixed it

Reply:
@rrawther Seems like this is still there